### DISCUSSION TITLE
Leap only works after having used Flit

### DISCUSSION THREAD

**Discussion by StephanRoemer** [LazyVim/LazyVim#495] [Q&A] [answered]
Not sure why that is, but when I press "s" in order to use Leap, nothing happens.
But, when using Flit and THEN Leap, then everything works as expected. Is this some new behavior?

**Comment by folke** [comment, marked as answer]
For me it works when just pressing `s`, so no that's not normal.

Did you change any settings of their settings?

When starting, what does `:verbose nmap s` say?

**Comment by StephanRoemer** [comment]
Hey Folke!

Damn, you are right, user error! I completely forgot to check my custom keymaps.lua that I copied over.
I had used Leap in my previous neovim installation and forgot to remove the duplicate keympapings.

Thanks as always!

Cheers,

Stephan
